src/go/k8s/controllers/redpanda/state.go

```diff
@@ -1,3 +1,12 @@
+// Copyright 2021 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
```

Comment:
Good that you added copyright 👍 

Why year 2021?

Reply:
Was not sure when this was originally created, thought it was around this time, but i can change it to today.